src/core/multifeaturelistmodelbase.cpp

```diff
@@ -386,6 +386,38 @@ bool MultiFeatureListModelBase::canMoveSelection() const
   return true;
 }
 
+bool MultiFeatureListModelBase::canRotateSelection() const
+{
+  if ( mSelectedFeatures.isEmpty() )
+    return false;
+
+  QgsVectorLayer *vlayer = mSelectedFeatures[0].first;
+  if ( !vlayer || vlayer->readOnly() || !( vlayer->dataProvider()->capabilities() & QgsVectorDataProvider::ChangeGeometries ) || vlayer->customProperty( QStringLiteral( "QFieldSync/is_geometry_locked" ), false ).toBool() || ( vlayer->geometryType() == Qgis::GeometryType::Point ) )
```

Comment:
Actually, as unintuitive as it might sound, points are valid geometries to rotate when you are dealing with multipoint, we should allow for it.